templates/web/base/admin/triage/_inspect.html

```diff
@@ -0,0 +1,117 @@
+[% permissions = c.user.permissions(problem) %]
```

Comment:
Again, worth factoring out some bits of _inspect.html into smaller templates so they can be shared? e.g. the top info bit